### DISCUSSION TITLE
Need help with $PROFILE location

### DISCUSSION THREAD

**Discussion by AlanW5510** [JanDeDobbeleer/oh-my-posh#4596] [Troubleshoot]
My company got Microsoft 365 and all of our document's folder are synced automatically. when I run $PROFILE in the terminal it points to my oneDrive and not my local documents. This broke Oh my Posh and I do not have the colored terminal anymore. I was trying to move the Powershell folder to C:\PowerShell but I cannot figure out how to get $PROFILE to point to that location. Any help would be appreciated/

**Comment by JanDeDobbeleer** [comment]
@AlanW5510 you can edit the $profile that PWSH loads, copy-pasting over the content from one profile to another will make it work.

**Reply by AlanW5510** [reply]
I just don't know how to do that.

**Reply by JanDeDobbeleer** [reply]
You can open the previous profile in a text editor, copy its contents and paste that in the current profile.

**Reply by AlanW5510** [reply]
So I moved my PowerShell folder back to documents and it technically works. But for some reason it is loading the wrong Theme. Here is what is in my Microsoft.PowerShell_profile.ps1 `oh-my-posh init pwsh --config "C:\PowerShell\kali.omp.json" | Invoke-Expression
Import-Module -Name Terminal-Icons`

The kali.omp.json is in the same directory. This is what my terminal looks like though and it switched to whatever this theme is when my documents got synced with OneDrive
![Terminal](https://github.com/JanDeDobbeleer/oh-my-posh/assets/23545052/03fde570-ef3a-4463-b38a-646672711593)
:

**Reply by AlanW5510** [reply]
It is working now. I just updated the path in the Microsoft.PowerShell_profile.ps1 to add the OneDrive location and it is working. Thank you for the help.
